app/gui2/src/components/DocumentationPanel.vue

```diff
@@ -128,8 +127,14 @@ function handleBreadcrumbClick(index: number) {
       @forward="historyStack.forward()"
       @backward="historyStack.backward()"
     />
-    <DocsTags v-if="sections.tags.length > 0" class="tags" :tags="sections.tags" />
+    <DocsTags
+      v-if="sections.tags.length > 0"
+      class="tags"
+      :tags="sections.tags"
+      :groupColor="color"
+    />
     <div class="sections">
+      <h2 v-if="documentation.kind === 'Placeholder'">{{ documentation.text }}</h2>
```

Comment:
i have no idea whether we have a convention to determine when to use `v-text` and when to use `{{}}`

Reply:
I have no idea either, but I find the current syntax more readable.